MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -36,6 +37,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Protocol/Security2.h>
 #include <Protocol/Reset.h>
 #include <Protocol/Cpu.h>
+#include <Protocol/Cpu2.h>           // MS_CHANGE

```

Comment:
`MS_CHANGE` should be `MU_CHANGE` in a couple of places

Reply:
Fixed.